src/change_stream/mod.rs

```diff
@@ -93,9 +95,11 @@ pub struct ChangeStream {
     /// The options provided to the initial `$changeStream` stage.
     options: Option<ChangeStreamOptions>,
 
-    /// The read preference for the initial `$changeStream` aggregation, used for server selection
-    /// during an automatic resume.
-    read_preference: Option<ReadPreference>,
+    /// The criteria used to select a server for this operation.
```

Comment:
I think this comment should be left as it was in the previous commit, just with "read preference" replaced with "selection criteria".

Reply:
fixed

Reply:
This doesn't seem to be fixed in the latest commit; the comment is still, the same as before. It should be the same as it was before this pull request (i.e. the left side of the diff in this view).

Reply:
didn't push earlier. fixed now